### PR TITLE
Paper-item is a button.

### PR DESCRIPTION
**Pull request by cdata**
This PR depends on https://github.com/PolymerElements/iron-menu-behavior/pull/25

### CODE REVIEW

File: paper-item.html [hunk 1]
```diff
@@ -76,8 +78,14 @@
       is: 'paper-item',
 
       hostAttributes: {
-        role: 'listitem'
-      }
+        role: 'listitem',
+        tabindex: '0'
```
Comment (notwaldorf) [inline paper-item.html]: i think now you can also fix the demo where i had to manually add `tabindex=0` to all the `paper-items`. Also, do you need it to `paper-icon-item` as well?

Comment (cdata) [inline paper-item.html]: Done.

**Comment by cdata** [comment]
Fixed paper-icon-item and removed tabindex from the demo. PTAL!

**Comment by notwaldorf** [comment]
:+1: :shipit:

**Pull request merged**